src/Service.Host/client/src/components/Workstations.js

```diff
@@ -22,18 +28,22 @@ function Workstations() {
             setCitCodeSearchTerm(newValue);
         }
     };
+    const auth = useAuth();
+    const token = auth.user?.access_token;
 
     const {
         send: getWorkStations,
         workStationsLoading,
         result: workStationsResult
-    } = useGet(itemTypes.workStations.url);
+    } = useGet(itemTypes.workStations.url, true);
 
     const [hasFetched, setHasFetched] = useState(false);
+    const [hasPermission, setHasPermission] = useState(false);
 
-    if (!hasFetched) {
+    if (!hasFetched && token) {
         setHasFetched(true);
         getWorkStations();
+        setHasPermission(getWorkStations(null, 'application-state'));
```

Comment:
does this work? what does getWorkStations return? I would assume nothing since all it does is fire off a request. To access the result you need to access the 'workStationsResult' value that the hook returns.

as an aside I would set up a seperate useGet specifically for the application state as opposed to trying to re-use the getWorkstations one